### DISCUSSION TITLE
Saving setup configurations to a file

### DISCUSSION THREAD

**Discussion by Aishima480** [ppy/osu#27656] [Ideas] [answered]
After each reinstallation of osu, you have to reconfigure everything manually, but this way, you can make it a lot easier. 
You don't have to do this for all sections. You can do it for at least the most important ones. For example: Tablet Settings, Key Assignment, Gameplay Settings.
Also, you can make a list of configurations if a person needs a few specific ones. I really ask you to consider adding such a feature as it is very useful.

**Comment by Stoppedpuma** [comment]
Most of the important configuration options are already being saved to files, check the game.ini and input.json files in your osulazer folder.

**Comment by peppy** [comment, marked as answer]
For now, you can just copy the game folder. It contains all configuration locally.

We will eventually add profiles for configuration.

**Reply by KaVoshnik** [reply]
Why not add a binding of the settings configuration to the player's profile? 

Conventionally, when Osu!Lazer is first launched on the device, the settings will be saved in a separate file that will be uploaded to the player's profile. You can also add a function to the settings menu to export/import settings from the player profile.

Thx in advance for reply!
